client/src/shared/ui/overlay/OverlayDropdown.js

```diff
@@ -162,8 +164,22 @@ function Option(props) {
     icon: IconComponent
   } = props;
 
+  const handleKeydown = (event) => {
+    const {
+      key,
+      keyCode,
+      currentTarget
+    } = event;
+
+    if (key === 'ArrowDown' || keyCode == 40) {
+      focusNext(currentTarget);
+    } else if (key === 'ArrowUp' || keyCode == 38) {
+      focusPrevious(currentTarget);
+    }
+  };
+
   return (
-    <li onClick={ onClick }>
+    <li role="menuitem" onClick={ onClick } onKeyDown={ handleKeydown }>
       <button type="button" title={ text }>
```

Comment:
Consider making the button the clickable element (for A11y purposes).

Reply:
Consider making the key-up-down-to-change selection a DOM-only / no react directive:

This would simplify it and allow us to re-use it in other areas easily.

Reply:
I actually come from this direction [in a first iteration](https://github.com/camunda/camunda-modeler/issues/2626#issuecomment-990714902) but went away from it. 

Simple reason: getting away from React made it [harder to test](https://github.com/enzymejs/enzyme/issues/441#issuecomment-441836305), firing the native events instead of simulating it in our existing Enzyme infrastructure.  Adding it via `onKeyDown` made it much easier to test.